Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine:3.17.1 AS builder
+ARG VERSION=0.110.0
+ADD https://github.com/gohugoio/hugo/releases/download/v0.110.0/hugo_extended_0.110.0_darwin-universal.tar.gz /hugo.tar.gz
+RUN tar -zxvf hugo.tar.gz && \
+    rm -rf hugo.tar.gz && \
+    apk add --no-cache git
```

Comment:
What purpose is this serving?

Reply:
On this we are pulling the HUGO environment image to our docker container so our site can run. That file we downloaded that is zip file so we are just doing extracting and removing the zip and git as well.